bindings/ios/MEGASdk.mm

```diff
@@ -1485,22 +1485,13 @@ - (void)setUploadLimitWithBpsLimit:(NSInteger)bpsLimit {
     self.megaApi->setUploadLimit((int)bpsLimit);
 }
 
-- (MEGABackgroundMediaUpload *)backgroundMediaUpload {
-    return [[MEGABackgroundMediaUpload alloc] initWithBackgroundMediaUpload:self.megaApi->backgroundMediaUploadNew()];
-}
-
-- (MEGABackgroundMediaUpload *)resumeBackgroundMediaUploadBySerializedData:(NSData *)data {
-    std::string serializedBytes = std::string((const char *)data.bytes, data.length);
-    return [[MEGABackgroundMediaUpload alloc] initWithBackgroundMediaUpload:self.megaApi->backgroundMediaUploadResume(&serializedBytes)];
-}
-
 - (void)requestBackgroundUploadURLWithFileSize:(int64_t)filesize mediaUpload:(MEGABackgroundMediaUpload *)mediaUpload delegate:(id<MEGARequestDelegate>)delegate {
-    return self.megaApi->backgroundMediaUploadRequestUploadURL(filesize, mediaUpload.getCPtr, [self createDelegateMEGARequestListener:delegate singleListener:YES]);
+    self.megaApi->backgroundMediaUploadRequestUploadURL(filesize, mediaUpload.getCPtr, [self createDelegateMEGARequestListener:delegate singleListener:YES]);
 }
 
-- (BOOL)completeBackgroundMediaUpload:(MEGABackgroundMediaUpload *)mediaUpload fileName:(NSString *)fileName parentNode:(MEGANode *)parentNode fingerprint:(NSString *)fingerprint originalFingerprint:(NSString *)originalFingerprint binaryUploadToken:(NSData *)token delegate:(id<MEGARequestDelegate>)delegate {
-    std::string binaryToken = std::string((const char *)token.bytes, token.length);
-    return self.megaApi->backgroundMediaUploadComplete(mediaUpload.getCPtr, fileName.UTF8String, parentNode.getCPtr, fingerprint.UTF8String, originalFingerprint.UTF8String, &binaryToken, [self createDelegateMEGARequestListener:delegate singleListener:YES]);
+- (void)completeBackgroundMediaUpload:(MEGABackgroundMediaUpload *)mediaUpload fileName:(NSString *)fileName parentNode:(MEGANode *)parentNode fingerprint:(NSString *)fingerprint originalFingerprint:(NSString *)originalFingerprint binaryUploadToken:(NSData *)token delegate:(id<MEGARequestDelegate>)delegate {
```

Comment:
Can we pass the thumbnails & preview to the `backgroundMediaUploadComplete`?
This way, the `putnode` will be done with the file attributes, and avoid at the app layer setting up the thumbnail and the preview after the `putnode`, right?

Reply:
Good question, since we have the discussion in Slack, let's move the discussion to Slack.

I have changed code according to new SDK changes. Now, we don't need to specify thumbnail & preview handles in put node.